.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: java
+
+env:
+  global:
+    - CXXFLAGS="--std=c++0x"
+
+install:
+  - buildscripts/make_dependencies.sh # build protoc into /tmp/grpc-deps
+
+script:
+  - export PATH=/tmp/grpc-deps/bin:$PATH
```

Comment:
I wasn't able to find a working example of doing $ expansions so I'm assuming this is necessary. Even Travis' own doc show this
http://docs.travis-ci.com/user/installing-dependencies/